lib/fastly/belongs_to_service_and_version.rb

```diff
@@ -23,8 +25,15 @@ def as_hash
       super.delete_if { |var| %w(service_id version).include?(var) }
     end
 
+    # URI escape (including spaces) the path and return it
+    def self.path_escape(path)
+      @uri_parser ||= URI::Parser.new
+      # the leading space in the escape character set is intentional
+      @uri_parser.escape(path, ' !*\'();:@&=+$,/?#[]')
```

Comment:
The `\'` section of this string will be interpreted by ruby as a literal quote character instead of both a backslash and a single quote.